deploy/canary/Dockerfile

```diff
@@ -1,10 +0,0 @@
-FROM golang:latest
```

Comment:
Yeah would be great to get rid of the canary.  If dockerhub automated builds can do a periodic build from master/head as well, then we can stop building it from jenkins.